extension/lib/main.js

```diff
@@ -6,11 +6,11 @@
 
```

Comment:
No need to hurry, those tests could land with #218 too with an another release! :)
Let have a working GUI first! :+1: